dao/src/integ-test/java/uk/gov/moj/sdt/dao/BulkSubmissionDaoTest.java

```diff
@@ -216,10 +222,13 @@ public void testGetBulkSubmissionPastRetention() {
         createBulkSubmission(customerReference,
                              LocalDateTime.now().minusDays(dataRetentionPeriod + 1),
                              sdtBulkReference);
-        final IBulkSubmission bulkSubmission =
+        IBulkSubmission bulkSubmission = null;
+        try {
+            bulkSubmission =
                 bulkSubmissionDao.getBulkSubmission(bulkCustomer, customerReference, dataRetentionPeriod);
+        } catch (Exception e) {
+        }
```

Comment:
Need to log the exception message here along with a message indicating where the failure occurred.  This will aid debugging if the test fails.